packages/react-components/src/components/Survey/StarSurvey/StarSurvey.tsx

```diff
@@ -84,6 +84,8 @@ export interface _StarSurveyProps {
   unselectedIcon?: string;
   /** Function to send StarSurvey results*/
   onSubmitSurvey?: (survey: _CallSurvey) => Promise<_CallSurveyResponse | undefined>;
+  /** function called on confirm button click*/
+  onConfirmStarSurvey?:(ratings:number) => void
```

Comment:
how does this differ to onSubmitSurvey?

Reply:
Onsubmitsurvey is what's called to submit the survey result, onConfirm is an optional callback users can pass in to do something when the confirm button is clicked